### PR TITLE
PIC-4149 use generic service helm chart

### PR DESCRIPTION
**Pull request by markreesmoj**

### CODE REVIEW

File: helm_deploy/values-prod.yaml [hunk 1]
```diff
@@ -1,44 +1,26 @@
...
 
-ingress:
-  enabled: true
-  enable_whitelist: true
-  hosts:
-    - host: crime-portal-mirror-gateway.service.justice.gov.uk
-      cert_secret: crime-portal-gateway-cert
-  path: /mirrorgateway/service/cpmgwextdocapi(/|$)(.*)
-  rewrite_target: /crime-portal-gateway/ws/$2
+  allowlist:
```
Comment (Obsiye) [inline helm_deploy/values-prod.yaml:15]: From what i see, we're removing IPs from the whitelist except CGI's ip ranges.

Would this block the /info and /health endpoints called by the HMPPS dashboard (tbf it's broken atm)

File: helm_deploy/values-dev.yaml [hunk 2]
```diff
@@ -1,34 +1,16 @@
...
-  office: "217.33.148.210/32"
-  health-kick: "35.177.252.195/32"
-  mojvpn: "81.134.202.29/32"
-  cloudplatform-live1-1: "35.178.209.113/32"
-  cloudplatform-live1-2: "3.8.51.207/32"
-  cloudplatform-live1-3: "35.177.252.54/32"
-  bsi-1: "54.37.241.156/30"
-  bsi-2: "167.71.136.237/32"
-  global-protect: "35.176.93.186/32"
+  allowlist:
```
Comment (Obsiye) [inline helm_deploy/values-dev.yaml:14]: i'm guessing if we want to send a payload to dev CPG, we'll just use the svc url e.g. https://serviceurl.com/

File: helm_deploy/crime-portal-gateway/values.yaml [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+      private-key-alias-name: PRIVATE_KEY_ALIAS_NAME
+
+  namespace_secrets_to_file:
+    crime-portal-gateway-keystore-cert:
+      /mnt/secrets:
+        - crime-portal-gateway-jks.key
+
+  allowlist:
+    groups:
+      - internal
```
Comment (Obsiye) [inline helm_deploy/crime-portal-gateway/values.yaml:56]: Curious where internal is configured?

Comment (markreesmoj) [reply helm_deploy/crime-portal-gateway/values.yaml:56]: https://github.com/ministryofjustice/hmpps-ip-allowlists/blob/main/ip-allowlist-groups.yaml#L26

Looks like it is deprecated, I now see

Comment (markreesmoj) [reply helm_deploy/crime-portal-gateway/values.yaml:56]: https://github.com/ministryofjustice/hmpps-ip-allowlists/tree/main?tab=readme-ov-file#hmpps-ip-allowlist-groups
This explains how it works in more detail

File: helm_deploy/values-dev.yaml (hunk 2 above)
Comment (markreesmoj) [reply helm_deploy/values-dev.yaml:14]: The `internal` range should mean we have the same access as we do now

File: helm_deploy/crime-portal-gateway/values.yaml (hunk 3 above)
Comment (markreesmoj) [reply helm_deploy/crime-portal-gateway/values.yaml:56]: https://github.com/ministryofjustice/hmpps-ip-allowlists/tree/main?tab=readme-ov-file#hmpps-ip-allowlist-groups

**Review by Obsiye — APPROVED** [review]

**Pull request merged**